integration/caching_test.go

```diff
@@ -168,12 +168,12 @@ func testCaching(t *testing.T, context spec.G, it spec.S) {
 				extenderBuildStr+"  Executing launch environment install process",
 				fmt.Sprintf(extenderBuildStr+"    Running 'npm ci --unsafe-perm --cache /layers/%s/npm-cache'", strings.ReplaceAll(settings.Buildpack.ID, "/", "_")),
 			))
-			Expect(logs).To(ContainLines(MatchRegexp(extenderBuildStrEscaped + `      Completed in (\d+\.\d+|\d{3})`)))
+			Expect(logs).To(ContainLines(MatchRegexp(extenderBuildStrEscaped + `      Completed in (\d+)(\.\d+)?(ms|s)`)))
```

Comment:
Just wondering what caused this change to be needed?

Reply:
No real reason. Just me being a bit frustrated with the unstable test. For one this doesn't test what it implies, but it just test that ANYWHERE in the test is a `Completed in` and that is not really meaningful, is it? But more important, the test failed if measured part took 100 - 999 ms, but faster. Having a test failing and debugging for that was a bit frustrating.

And the same assertion is done all over the place, only here it is a bit different. But if you prefer, I can of course revert it. But it shouldn't cause any problems. Should it?